Tester/test_checkfilecheckbuttonpressed.cpp

```diff
@@ -38,3 +38,55 @@ pressTest.checkStatusOfButtonPressed();
 CHECK(pressTest.exit == true);
 
 }
+
+TEST_CASE( "check button pressed ADD", "[checkButtonPressed]" ) {
+
+CheckButtonPressed pressTest;
+
+keybd_event(VK_ADD, 0, KEYEVENTF_EXTENDEDKEY, 0);
+keybd_event(VK_ADD, 0, KEYEVENTF_EXTENDEDKEY | KEYEVENTF_KEYUP, 0);
+pressTest.checkStatusOfButtonPressed();
+
+CHECK(pressTest.getHowManySec() == 2);
```

Comment:
This test checks if key press was detected, or that key press changes some values?